src/NumericalAlgorithms/Interpolation/InterpolatedVars.hpp

```diff
@@ -115,5 +115,23 @@ struct HolderTag {
              typename InterpolationTargetTag::vars_to_interpolate_to_target>;
 };
 
+/// Indexes a list of interpolation points in the `TaggedTuple` that
+/// is accessed from the `Element`'s `DataBox` with tag
+/// `Tags::InterpPointInfo`.
+///
+/// This is for the case in which interpolation bypasses the
```

Comment:
This tag is

Reply:
can this be put in a separate header to reduce dependencies...

Reply:
I'm not sure how you want to reduce dependencies.  Do you want `Tags::InterpPointInfo` and `Vars::PointInfoTag` to be both put into a new header, e.g. PointInfoTag.hpp ?

Reply:
Well it looks like `PointInfoTag` doesn't depend on anything else in this file, and is what is used in the element, so just moving that to a separate file would reduce the amount of unnecessary dependencies, right?